samza-yarn/src/main/java/org/apache/samza/job/yarn/YarnClusterResourceManager.java

```diff
@@ -120,6 +119,11 @@ public YarnClusterResourceManager(Config config, JobModelManager jobModelManager
     hConfig = new YarnConfiguration();
     hConfig.set("fs.http.impl", HttpFileSystem.class.getName());
 
+    if (config.containsKey("fs.certfs.impl.override")) { //TODO: change to use constants from CertFSConstants once that CertFSConstants.java is in
```

Comment:
Perhaps, I'm missing something. Why do we need 2 configs - one for `fs.certfs.impl` and the `override` config?